tests/data/docstring.py

```diff
@@ -136,3 +143,8 @@ def over_indent():
       - And the closing quote is too deep
     """
     pass
+
+
+def single_line():
+    """But with a newline after it!"""
+    pass
```

Comment:
Is this an accidental duplicate of the one above at lines 69 to 75? It is refining the function...

Reply:
Nevermind, didn't see the output comment.